app/src/main/java/com/example/concordia_campus_guide/LocationFragment/LocationFragment.java

```diff
@@ -256,7 +258,7 @@ private boolean requestPermission(){
 
     @Override
     public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
-
+        initMap();
```

Comment:
Why are we initializing the map at both line 261 and 66?

Reply:
I added it by mistake n forgot to remove it.